STORAGE.md

```diff
@@ -183,6 +183,8 @@ http://104.131.175.196:8080/, http://70.88.180.169:8088/, http://207.237.122.161
 * ⭐ **[GeeksforGeeks](https://www.geeksforgeeks.org/)**
 * ⭐ **[W3Schools](https://www.w3schools.com/)** - [Offline Version](https://github.com/Ja7ad/W3Schools)
 * ⭐ **[Build your own X](https://github.com/codecrafters-io/build-your-own-x)**
+* ⭐ **[TakeuForward](https://takeuforward.org)**  DSA / SDE / Roadmap

```

Comment:
Use `-` to delimit these, see the above items for example.

Reply:
> Use `-` to delimit these, see the above items for example.

used - to delimit